server/openslides/poll/serializers.py

```diff
@@ -136,7 +141,17 @@ def validate(self, data):
             )
         return data
 
-    def norm_100_percent_base_to_pollmethod(
-        self, onehundred_percent_base, pollmethod, old_100_percent_base=None
+    def norm_100_percent_base(
+        self, onehundred_percent_base, pollmethod, polltype, old_100_percent_base=None
     ):
-        raise NotImplementedError()
+        if (
```

Comment:
Looks  like this needs to be ported to OS4 as well.

Reply:
Yes, I'll open issues for it